.github/workflows/ci-pr.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       shell: bash
       timeout-minutes: 75
       run: |
-        ./mvnw -B -s .github/settings.xml -Pdocs clean install --no-transfer-progress
+        ./mvnw -B -s .github/settings.xml -Pfull,docs clean install --no-transfer-progress
```

Comment:
What is `full` vs. `docs`? I guess I don't understand how `full` fulfills the PR description of `executes rest-docs generation as well`.

Reply:
I didn't choose the name. We can change to `rest-docs` to ensure it is unambiguous.

Reply:
Having checked:
* `docs` means javadocs
* `full` means all rest-docs and asciidocs.

How about we rather use `javadocs` and `fulldocs`?

Reply:
Makes sense to me .   @onobc what do you think?